src/script_opt/CPP/maint/find-test-files.sh

```diff
@@ -1,7 +1,7 @@
 #! /bin/sh
 
 find ../testing/btest -type f |
-    grep -E -v 'Baseline|\.tmp' |
+    grep -E -v 'Baseline|\.tmp|__load__' |
```

Comment:
1. Yes, the goal is to find all of the BTests
2. All it needs - or wants - is the top-level script. The way we do our BTests, those are `.zeek` or `.test` or such, rather than directories driven off of `__load__.zeek`s inside them
3. Such loads do occur, but in the scripts directories for BTests that have plugins. For those we _don't_ want the script (which is why this wound up being added), since it's not itself a BTest
4. Great suggestion re `@TEST`! I'll put that on the maintenance list